drawing_5.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<head>
+<html lang='en'>
+	<meta charset="UTF-8">
+		<title>Drawing 5</title>
+			<link rel="stylesheet" type="text/css" href="drawing_5.css">
+</head>
+
+<body>
+	<div class="one">
+		<div class="single"></div>
+	</div>
+	<div class="leftcon">
```

Comment:
You need to be careful about your naming. Your other `divs` are `one` `three` and `five`. What happened to `two` and `four`. These missing numbers would confuse another programmer. So either make the big `divs` - `one, two, three`, or make `leftcon` and `rightcon` into `two` and `four`.